src/app/components/hero-gallery.tsx

```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+
+export default function HeroGallery() {
+  return (
+    <div className="container">
+      <div className="center_img">
+        <Image
+          className="img--header"
+          src="/home-page-gallery/ctp-render-1.jpg"
+          width={500}
+          height={500}
+          alt="Exterior rendering of an accomodation architectural project"
+        />
+        <div>
+          <Image
+            src="/home-page-gallery/instagram-logo.png"
+            width={30}
+            height={30}
+            alt="instagram-logo"
+            style={{ padding: 20 }}
```

Comment:
maybe it would be a good idea to have the styling separated in case you need to reuse it for another social icon. that's more of a good practice

Reply:
There is an annoying thing about the <Image> tag.  It is mandatory to provide a width and height, even if you apply a class that overwrites these two props. About the padding, yes, you are right. I don't usually use in line styling, but I thought this was an isolated case and the probability of using this exact logo and style somewhere else is very low.